roles/irods/tasks/main.yml

```diff
@@ -85,16 +94,21 @@
       - "{{ irods_database_pkgname }}"
   when: ansible_os_family == 'RedHat'
 
-- name: Ensure iRODS server packages are version-locked (Ubuntu)
+- name: Ensure iRODS server packages are version-locked (Ubuntu / Debian)
   ansible.builtin.dpkg_selections:
     selection: hold
-    name:
-      - "{{ irods_server_pkgname }}"
-      - "{{ irods_runtime_pkgname }}"
-      - "{{ irods_icommands_pkgname }}"
-      - "{{ irods_database_pkgname }}"
+    name: "{{ item }}"
```

Comment:
Hi Bruno,

why do you use a loop for this one?
It's normally faster to group packages under the `name` attribute. (only one `dpkg_selections` call).

Reply:
Because it fails with Debian 12:

```
fatal: [mesonet-sto1.univ-lyon1.fr]: FAILED! => {
    "changed": false,                                                                                         
    "cmd": "/usr/bin/dpkg --set-selections",
    "invocation": {                                                                                           
        "module_args": {
            "name": "['irods-server', 'irods-runtime', 'irods-icommands', 'irods-database-plugin-postgres']",
            "selection": "hold"
        }
    },
    "msg": "dpkg: error: unexpected data after package and selection at line 1",                              
    "rc": 2,                                                                                                  
    "stderr": "dpkg: error: unexpected data after package and selection at line 1\n",                         
    "stderr_lines": [
        "dpkg: error: unexpected data after package and selection at line 1"
    ],
    "stdout": "",
    "stdout_lines": []
}
```

Reply:
OK. Fair enough.